app/controllers/web-payments/apple-pay/merchant-validation.controller.js

```diff
@@ -66,92 +63,59 @@ module.exports = async (req, res) => {
-      body: {
-        merchantIdentifier: merchantIdentityVars.merchantIdentifier,
-        displayName: 'GOV.UK Pay',
-        initiative: 'web',
-        initiativeContext: process.env.APPLE_PAY_MERCHANT_DOMAIN
-      },
-      json: true
-    }
-
-  if (applePayMerchantValidationViaAxios) {
-    if (proxyUrl) {
-      logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent)')
-
-      const data = {
-        merchantIdentifier: merchantIdentityVars.merchantIdentifier,
-        displayName: 'GOV.UK Pay',
-        initiative: 'web',
-        initiativeContext: process.env.APPLE_PAY_MERCHANT_DOMAIN
-      }
-
-      try {
-        const response = await axiosInstance.post(url, data, { headers: { 'Content-Type': 'application/json; charset=utf-8' } })
+  const options = {
+    url: url,
+    method: 'post',
+    cert: merchantIdentityVars.cert,
+    key: merchantIdentityVars.key,
+    headers: { 'Content-Type': 'application/json' },
+    data: {
+      merchantIdentifier: merchantIdentityVars.merchantIdentifier,
+      displayName: 'GOV.UK Pay',
+      initiative: 'web',
+      initiativeContext: process.env.APPLE_PAY_MERCHANT_DOMAIN
+    },
+    httpsAgent: httpsAgent
+  }
 
-        logger.info('Apple Pay session successfully generated via axios and https proxy agent (hpagent)')
-        res.status(200).send(response.data)
-      } catch (error) {
-        logger.info('Error generating Apple Pay session with axios and https proxy agent (hpagent)', {
-          ...getLoggingFields(req),
-          error: error.message,
-          status: error.response ? error.response.status : 'No status'
-        })
-        logger.info('Apple Pay session via axios and https proxy agent (hpagent) failed', 'Apple Pay Error')
-        res.status(500).send('Apple Pay Error')
-      }
-    } else {
-      logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent) (NO PROXY)')
-      try {
-        const response = await axios(options)
+  if (proxyUrl) {
+    logger.info('Generating Apple Pay session via axios and https proxy agent (hpagent)')
 
-        logger.info('Apple Pay session successfully generated via axios and https proxy agent (hpagent) (NO PROXY)')
-        res.status(200).send(response.data)
-      } catch (error) {
-        logger.info('Error generating Apple Pay session (NO PROXY)', {
-          ...getLoggingFields(req),
-          error: error.message
-        })
-        logger.info('Apple Pay session via axios and https proxy agent (hpagent) with no proxy failed', 'Apple Pay Error')
-        res.status(500).send('Apple Pay Error')
-      }
+    const data = {
+      merchantIdentifier: merchantIdentityVars.merchantIdentifier,
+      displayName: 'GOV.UK Pay',
+      initiative: 'web',
+      initiativeContext: process.env.APPLE_PAY_MERCHANT_DOMAIN
     }
 
+    try {
+      const response = await axiosInstance.post(url, data, { headers: { 'Content-Type': 'application/json; charset=utf-8' } })
```

Comment:
In theory we shouldn't need to use a separate axios instance, doing `axios.post(url, data, config)` should work, so long as the `config` object has the `httpsAgent` set as the proxy agent and `proxy: false` set.

Reply:
Maybe something to leave for a later PR though so it can be tested separately